pkg/upgrade/upgrade_state_test.go

```diff
@@ -55,6 +55,68 @@ var _ = Describe("UpgradeStateManager tests", func() {
 		stateManager.ValidationManager = &validationManager
 
 	})
+
```

Comment:
Have you added tests for upgrade requested label ? (move to upgrade required and the removal of the label ?)

Reply:
Done